web/app/js/components/LineGraph.jsx

```diff
@@ -133,6 +135,40 @@ export default class LineGraph extends React.Component {
     }
   }
 
+  flashLatestDataPoint() {
+    this.svg.select("circle").remove();
+    let circleData = {};
+
+    if (!_.isEmpty(this.props.data) && this.props.showFlash) {
+      circleData = this.props.data.slice(-1)[0];
+      this.svg.append("circle")
+        .attr("class", "flash")
+        .style("opacity", 0.6)
+        .attr("cx", () => {
+          if (circleData.timestamp) {
+            return this.xScale(circleData.timestamp);
+          }
+        })
+        .attr("cy", () => {
+          if (circleData.value) {
+            return this.yScale(circleData.value);
+          }
+        })
+        .attr("r", 6)
+        .transition()
+        .on("start", function repeat() {
+          d3.active(this)
+            .transition()
+            .duration(550)
```

Comment:
(realizing we don't consistently do this elsewhere), but consider putting these magic numbers into constants or at least variables.